Jellyfin.Plugin.Bookshelf/Providers/GoogleBooks/GoogleBooksImageProvider.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using MediaBrowser.Common.Net;
+using MediaBrowser.Controller.Entities;
+using MediaBrowser.Controller.Providers;
+using MediaBrowser.Model.Entities;
+using MediaBrowser.Model.Providers;
+using MediaBrowser.Model.Serialization;
+using Microsoft.Extensions.Logging;
+
+namespace Jellyfin.Plugin.Bookshelf.Providers.GoogleBooks
+{
+    public class GoogleBooksImageProvider : IRemoteImageProvider
+    {
+        private static IHttpClient _httpClient;
+        private static IJsonSerializer _jsonSerializer;
+        private static ILogger _logger;
+
+        public GoogleBooksImageProvider(ILogger logger, IHttpClient httpClient, IJsonSerializer jsonSerializer)
+        {
+            _httpClient = httpClient;
+            _jsonSerializer = jsonSerializer;
+            _logger = logger;
+        }
+
+        public string Name => "Google Books";
+
+        public bool Supports(BaseItem item)
+        {
+            return item is Book;
+        }
+
+        public IEnumerable<ImageType> GetSupportedImages(BaseItem item)
+        {
+            return new List<ImageType>
```

Comment:
I don't think returning a `List` is the best solution, `yield return` is better IMO (or maybe even a plain array).